### DISCUSSION TITLE
Add dark mode support for sites which use `[data-theme="dark"]`

### DISCUSSION THREAD

**Discussion by vaibhavshn** [tailwindlabs/tailwindcss#5596] [Help] [answered]
So right now tailwindcss has support for dark mode via `classes` and `media queries`. I'm suggesting adding one more option to this, which would be for using tailwind's dark mode with sites that use `[data-theme="dark"]`.

This feature request came about as I was working on a docusaurus site which uses the data-theme method of implementing dark mode and I couldn't use tailwind's dark mode features. So I had to use CSS vars and set colors for both light and dark mode in the config file, which wasn't very intuitive.

Benefits:
- Tailwind will support more types of dark mode handling among websites.
- Less hassle for implementing dark mode in such sites

**Comment by oliverpool** [comment, marked as answer]
It looks like this is supported in the current tailwind version: https://tailwindcss.com/docs/dark-mode#customizing-the-class-name

**Reply by vaibhavshn** [reply]
Yes, I've used it. I'll close this now.

**Reply by vaibhavshn** [reply]
How do I mark this as closed/complete?

**Reply by oliverpool** [reply]
I think you can mark my answer as "correct"
Like in https://github.com/tailwindlabs/tailwindcss/discussions/10028
